spec/models/pdf_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Pdf, type: :model do
+  it "should be valid with a pdf_file and has a paperclip attachment" do
+    pdf = create(:pdf)
+
+    expect(pdf).to be_valid
+    expect(pdf.pdf_file.class).to eql(Paperclip::Attachment)
+    expect(pdf.pdf_file.url).to match(/public\/assets\/pdfs\/sample_pdf.pdf/)
```

Comment:
the pdf_file.url is set in the paperclip configs in the pdf model. Basically the .url method determines where this file is downloaded. If you change the :path, you need to change the :url in paperclip.... I think.